### ISSUE TITLE
Angular-Storybook: a11y addOn does not show violations when it should do

### ISSUE THREAD

**Issue by jameshughes7** [storybooks/storybook#5774]
**Describe the bug**
When using the a11y storybook addon, scenarios which should return violations but are not doing so.

**To Reproduce**
1. Visit [Storybook official netlify link](https://next--storybooks-official.netlify.com/?path=/story/addons-a11y-basebutton--invalid-contrast)
2. Note how there are  **_No a11y violations found_** with the **Invalid contrast** example
3. Attempting this myself within my own project I'm seeing similar results:
4. Configure angular project to use **a11y addon** by following the [getting started doc](https://github.com/storybooks/storybook/tree/master/addons/a11y#getting-started):
  - `npm install -D @storybook/addon-a11y`
  - `import '@storybook/addon-a11y/register';`
5. Configure stories with **a11y addon**: 
  - `import { checkA11y, configureA11y } from '@storybook/addon-a11y';`
6. Specify stories file with a11y configurations 
7. Add a11y decorator to stories file
8. Set the following scss attributes of our button to the color white: background-color, background-color-hover and background-color-active.
9. Set the display attribute of our component to `display: none`

**Expected behavior**
Should see violations as component is not meeting the a11y configurations `['wcag2a', 'wcag2aa', 'wcag21aa']`

**Screenshots**
**Storybook official netlify link**
<img width="1187" alt="storybook_a11y_no_violations_bug" src="https://user-images.githubusercontent.com/30200120/53411253-ec71e480-39bd-11e9-9ba9-67cc531ad994.png">

**My own project**
<img width="562" alt="own_project_a11y_no_violations_bug" src="https://user-images.githubusercontent.com/30200120/53411678-0a8c1480-39bf-11e9-843d-b4b14bd32f5d.png">

**Code snippets**

**button.stories.ts**
```
import { storiesOf } from '@storybook/angular';
import { withNotes } from '@storybook/addon-notes';
import { action } from '@storybook/addon-actions';
import { checkA11y, configureA11y } from '@storybook/addon-a11y';

import { ButtonComponent } from '../projects/component-library/src/lib/buttons/button/button.component';

// a11y Configurations
const a11yConfigs = {
  runOnly: ['wcag2a', 'wcag2aa', 'wcag21aa'],
};
configureA11y(a11yConfigs);

storiesOf('Buttons/Button', module)
  .addDecorator(checkA11y)
  .addDecorator(withNotes)
  .add(
    'Primary',
    () => ({
      component: ButtonComponent,
      props: {
        btnClick: action('Button was clicked'),
        text: 'Prescribe',
      },
    }),
    { notes: 'Primary style button' }
  )
  .add(
    'Primary disabled',
    () => ({
      component: ButtonComponent,
      props: {
        text: 'Prescribe',
        disabled: true,
      },
    }),
    { notes: 'Primary style disabled button' }
  );
```

**button.component.scss**
```
$primary-color: #0033a0;
$primary-hover: #00277c;
$primary-disabled: #cac8c5;

button {
  display: flex;
  align-items: center;
  justify-content: center;
  cursor: pointer;
  height: 44px;
  letter-spacing: 0.01em;
  font-size: 14px;
  border-radius: 100px;
  min-width: 196px;
  padding: 0 16px;
  font-family: Helvetica;
  color: #fff;

  &:focus {
    outline: none;
  }

  &.primary {
    background-color: #fff;

    &:hover {
      background-color: #fff;
    }

    &:active {
      background-color: #fff;
    }

    &:disabled {
      cursor: default;
      background-color: $primary-disabled;
    }
  }
}
```

**System:**
 - OS: [MacOS]
 - Device: [Macbook Pro 2017]
 - Browser: [chrome]
 - Framework: [angular]
 - Addons: [a11y]
 - Version: [5.0.0-rc.5]

**Additional context**
n/a

**Comment by gaetanmaisse** [comment]
Storybook is not displaying any violations because in this case (BaseButton | Invalid contrast) `axe-core`, the library we are using for a11y, doesn't return any violations.

In fact this `color-contrast` rule is not in "passing" rules array neither but in `incomplete`. 
![image](https://user-images.githubusercontent.com/4112568/53444999-d68d0f80-3a0f-11e9-81c4-e8e178c5d2c3.png)


 As [per doc](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#results-object
): 

> The incomplete array (also referred to as the "review items") indicates which nodes could neither be determined to definitively pass or definitively fail. They are separated out in order that a user interface can display these to the user for manual review (hence the term "review items").

BUT for now only `passes` and `violations` array are displayed in SB ui. Maybe we should think about adding something to display `incomplete` rules array?

**Comment by shilman** [comment]
Thanks @gaetanmaisse . Opened a new feature request for this and closing this issue for now.
